### PR TITLE
Limit on the number of aggregatable reports per source

### LINKED ISSUE #740: Consider a maximum # of reports per source for aggregatable sources
This is related to the cross-site information leakage discussed in https://github.com/WICG/attribution-reporting-api/issues/738 and https://github.com/WICG/attribution-reporting-api/issues/439. If an adversary triggers many reports for a given source, the protections we are thinking about degrade.

One way to help this situation would be to set a limit. As a strawman, I am thinking of something like 20 aggregatable reports per source, but feedback welcome.

### PR DESCRIPTION
**Pull request by akashnadan**
Fixes #740

### CODE REVIEW

**Review by csharrison — APPROVED** [review]
Looks like you added a bunch of unrelated newline changes. Can you revert those

File: AGGREGATE.md [hunk 1]
```diff
@@ -387,6 +387,7 @@ other representing the conversion dollar value per geography. This budgeting
 mechanism is highly flexible and can support many different aggregation
 strategies as long as the appropriate scaling is performed on the outputs. 
 
+
```
Comment (apasel422) [inline AGGREGATE.md:390]:
```suggestion
```

File: AGGREGATE.md [hunk 2]
```diff
@@ -504,6 +507,7 @@ Note: there are a few caveats about a formal differential privacy claim:
   [event-level explainer](https://github.com/WICG/attribution-reporting-api/blob/main/EVENT.md#reporting-cooldown--rate-limits)
   should also apply to aggregatable reports. The limits should be shared across
   all types of reports.
+
```
Comment (apasel422) [inline AGGREGATE.md:510]:
```suggestion
```

**Review by apasel422 — CHANGES_REQUESTED** [review]

**Review by apasel422 — APPROVED** [review]

**Comment by apasel422** [comment]
> Looks like you added a bunch of unrelated newline changes. Can you revert those

Done.

File: AGGREGATE.md [hunk 3]
```diff
@@ -430,6 +430,8 @@ Strawman: There should be ~0.05 reports (in expectation) sent for trigger
 registrations that exclude the `source_registration_time` field, and ~0.25 reports for
 those that include this field.
 
+In order to limit abuse of the protections above, there will be a maximum limit of 20 aggregatable reports per source.
```
Comment (apasel422) [inline AGGREGATE.md:433]:
```suggestion
In order to limit abuse of the protections above, there will be a limit of 20 aggregatable reports per source.
```

**Comment by apasel422** [comment]
Please use a more descriptive PR title.

**Pull request merged**